shared/src/vol_interface.c

```diff
@@ -271,7 +258,6 @@ DllExport bool native_vol_open_video_file(const char* filename)
 #endif
     if ( ret ) {
         vol_av_dimensions( &video_file_ptr, &vid_w, &vid_h );
```

Comment:
i removed reading the frame rate at opening time and instead call it on request, and i also used it in the player code.

* not every (older) vologram has 30fps, some had 29.9 or whatever, so may as well check and use that in case we change the fps in the future (we might).
* the vol_av code returns 30 if it cant find the fps i believe, so should be safe...ish.
* fps can actually change during a video (not our ones but in general), so may as well not rely on the value given at opening.